clients/privacy-center/components/modals/VerificationForm.tsx

```diff
@@ -86,13 +91,19 @@ const useVerificationForm = ({
         const data = await response.json();
 
         if (!response.ok) {
-          handleError(data?.detail);
+          handleError({
+            title: "An error occurred while verifying your identity",
+            error: data?.detail,
```

Comment:
should we still have the `fallbackMessage` here?

Reply:
I think it's okay to leave it out here. The title is very similar to the original `fallbackMessage`. I'm using the error field here for more specific responses from the backend.

The previous version wasn't doing a title and error message at the same time. So the title is acting as a fallback already and the error message is only used if more data is provided from the backend.